### ISSUE TITLE
Installing Tailscale RPM on Fedora Silverblue breaks gnome-software

### ISSUE THREAD

**Issue by vega-d** [tailscale/tailscale#5582]
### What is the issue?

Well firstly, installing tailscale on Silverblue is a hassle, because the install script tries to use `dnf` which is not present.

However, even after successfully installing tailscale gui app store `gnome-software` fails to acquire metadata lock file (seemingly) due to some kind of a misconfiguration of repository's gpg key. 

this results in a complete breakage of gnome software (but not the cli updater, `rpm-ostree`) which I'd say is a pretty big deal on a desktop operating system. 

![image](https://user-images.githubusercontent.com/35414314/189330810-616209ae-730a-470d-9ee3-90e1f69897be.png)

### Steps to reproduce

First, the process of adding the repository is a little different.
```
sudo wget https://pkgs.tailscale.com/stable/fedora/tailscale.repo -P /etc/yum.repos.d/
```

(`--apply-live` flag will make it so applying the new package won't require a reboot)
```
sudo rpm-ostree install --apply-live tailscale
```
at that point, `gnome-software` will already error out and refuse to work. 

### OS

Linux

### OS version

Fedora Linux 36.20220908.0 (Silverblue) x86_64

### Tailscale version

1.30.0

### Bug report

BUG-90ab39504ba55ea19bb5a7e3da110ae5e5ee75e5a6556cb767643e819b5f5c6e-20220909103650Z-f144a7c2e79d614c

**Comment by vega-d** [comment]
After revisiting this, I figured out that gnome-software seems to be failing to verify the .repo gpg key for tailscale. If I disable `gpgcheck` and `repo_gpgcheck` inside the .repo file, everything works fine. 

![image](https://user-images.githubusercontent.com/35414314/199205239-77a7db76-141f-4748-9cf2-0a79634c27a7.png)

On one hand, gnome-software probably should handle this better than giving out non-descriptive permission denied errors, on the other hand gpg key for tailscale really does appear to not work, not sure, maybe I'm testing it wrong.

**Comment by bronson** [comment]
Outstanding investigative work. Turning off gpg fixed it for me too.

Tailscale definitely needs to fix this repo file, and I've opened an issue on gnome-software to hopefully do better than showing a broken UI: https://gitlab.gnome.org/GNOME/gnome-software/-/issues/1974

**Comment by ianloic** [comment]
It would be great to get separate instructions on the download page for Fedora Silverblue: https://tailscale.com/download/linux/fedora
and probably a version of https://pkgs.tailscale.com/stable/fedora/tailscale.repo that has `*gpgcheck=0`

**Comment by DentonGentry** [comment]
> Tailscale definitely needs to fix this repo file

I do not understand what is supposedly wrong with the repo file.

**Comment by sanecito** [comment]
This isn't an issue with Tailscale. I came across this issue with other packages and it ultimately boils down to GNOME Software, not Tailscale. The issue itself is patched with GNOME 44: https://gitlab.gnome.org/GNOME/gnome-software/-/issues/1793

I was able to install Tailscale today w/ an up to date version of Fedora Silveblue 38 (which has GNOME 44) and still update packages, and update to the latest OSTree commit later in the day when it came out.

This issue should be closed unless the ticket opener still has the issue on an up to date version of Fedora Silverblue
